srcpkgs/libgedit-gtksourceview/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'libgedit-gtksourceview'
+pkgname=libgedit-gtksourceview
+version=299.0.5
+revision=1
+build_helper="gir"
+build_style=meson
+configure_args="-Dgtk_doc=false"
+hostmakedepends="pkg-config glib-devel gettext"
+makedepends="gtk+3-devel libxml2-devel"
+short_desc="Source code editing widget"
+maintainer="Matt Boehlke <mtboehlke@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://gedit-technology.github.io"
+changelog="https://raw.githubusercontent.com/gedit-technology/libgedit-gtksourceview/main/NEWS"
+distfiles="https://github.com/gedit-technology/libgedit-gtksourceview/releases/download/${version}/libgedit-gtksourceview-${version}.tar.xz"
+checksum=4bdac3c6dd885a2af8064a7265618ff8505b2324ab02fb00b0ce55e02978d3d6
+make_check=no # Needs X
```

Comment:
```suggestion
make_check_pre="xvfb-run"
```

I tested this and the checks pass when building for x86_64-glibc, let's see if it passes in CI.

Reply:
@cinerea0 Thanks for the suggestions!  We'll see what happens in the CI.  On my local machine (i686), I had one test  time out .